python_modules/dagster/dagster_tests/daemon_tests/conftest.py

```diff
@@ -1,18 +1,25 @@
 import os
 import sys
+from typing import Generator, Optional, cast
 
 import pytest
-from dagster._core.host_representation import InProcessRepositoryLocationOrigin
+from dagster import DagsterInstance
+from dagster._core.host_representation import (
+    ExternalRepository,
+    InProcessRepositoryLocationOrigin,
+    RepositoryLocation,
+)
 from dagster._core.test_utils import (
     InProcessTestWorkspaceLoadTarget,
     create_test_daemon_workspace_context,
     instance_for_test,
 )
 from dagster._core.types.loadable_target_origin import LoadableTargetOrigin
+from dagster._core.workspace.context import WorkspaceProcessContext
 
 
 @pytest.fixture(name="instance_module_scoped", scope="module")
-def instance_module_scoped_fixture():
+def instance_module_scoped_fixture() -> Generator[DagsterInstance, None, None]:
```

Comment:
Prefer `Iterator[Foo]` to `Generator[Foo, None, None]`-- also applies to other instances in this PR